core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateRangeTests.java

```diff
@@ -56,6 +57,9 @@ protected DateRangeAggregatorBuilder createTestAggregatorBuilder() {
         if (randomBoolean()) {
             factory.missing(randomIntBetween(0, 10));
         }
+        if (randomBoolean()) {
+            factory.timeZone(DateTimeZone.forOffsetHours(randomIntBetween(-23, 23)));
```

Comment:
Not such a good idea, apparently Java TimeZone has some IDs which joda doesn't understand, so I'll stick with the zone IDs joda provides.